ripe/atlas/tools/commands/probes.py

```diff
@@ -166,6 +166,15 @@ def add_arguments(self):
                 "command."
             )
         )
+        self.parser.add_argument(
+            "--status",
+            type=int,
+            choices=[0, 1, 2, 3],
```

Comment:
It would be nice to have the option to say `--status connected` rather than `--status 1`.  As it is, this isn't very user-friendly.